src/main/webapp/js/components/interface/drawer/TabbedDrawer.less

```diff
@@ -0,0 +1,142 @@
+@import "./../../../../style/less/components/colors";
+
+@-webkit-keyframes fadeIn {
+  from {
+      opacity: 0;
+  }
+  to {
+      opacity: 1;
+  }
+}
+
+@keyframes fadeIn {
+  from {
+      opacity: 0;
+  }
+  to {
+      opacity: 1;
+  }
+}
+
+.geppettoDrawer {
+display: inline-block;
+width: 100%;
+height: 100%;
+max-height : ~"calc(100% - 40px)";
+}
+
+.tabber {
+  -webkit-animation: fadeIn 1s;
+  animation: fadeIn 1s;
+  border-bottom: 0.5px solid @primary_color;
+  width: 100%;
+  height: 35px;
+  display: block;
+  margin-top: 0px;
+  text-decoration: none;
+  left: 0;
+  bottom: 0;
+  position: absolute;
+  background: transparent;
+}
+
+
+.icons {
+  color: @primary_color;
+  background-color: @background_color_widget;
+  border:1px solid @primary_color;
+  position: absolute;
+  vertical-align:middle;
+  text-align: center;
+  bottom: 0;
+  height: 15px;
+  width: 15px;
+  cursor: pointer;
+  border-radius: 2px;
+  animation: fadeIn 1s;
+  -webkit-animation: fadeIn 1s;
+  transition: opacity 1s ease-out;
+}
+
+.icons:hover {
+  color: @background_color_widget;
+  background-color: @primary_color;
+  border:1px solid @primary_color;
+}
+
+.icons.closeIcons {
+  right: 0px;
+  margin-left: 5px;
+}
+
+.icons.maxIcons {
+  right: 20px;
+  margin-left: 5px;
+}
+
+.icons.minIcons {
+  right: 40px;
+}
+
+.tabButton {
+  color: @background_color_widget;
+  background-color: @primary_color;
+	margin: 0 auto;
+	margin-bottom: 26px;
+  position: relative;
+  border-color: @primary_color;
+  border: 0.5px;
+  border-bottom: 0px;
+	border-style: solid;
+	box-shadow: none;
+	text-shadow: none;
+	width: 140px;
+	height: 35px;
+  padding: 7px 20px;
+  text-align: center;
+  display: inline-block;
+  bottom: 0;
+  cursor: pointer!important;
+}
+
+.tabButton:hover {
+  color: @primary_color;
+  background-color: @background_color_widget;
+  border-color: @primary_color;
+  opacity: 0.8;
+}
+
+.buttonSelected {
+  color: @primary_color;
+  background-color: @background_color_widget;
+  border-color: @primary_color;
+  opacity: 0.8;
+}
+
+.drawer {
+  display: none;
+  position: absolute;
+  bottom: 0px;
+  left: 0px;
+  height: 250px;
+  width: 100%;
+  animation: fadeIn 1s;
+  -webkit-animation: fadeIn 1s;
+  transition: opacity 1s ease-out;
+  background-color: @background_color_widget;
+  background: @background_color_widget;
+  color: white;
+  opacity: 0.8;
```

Comment:
I know you haven't changed this but relooking at it it seems a bit difficult to read, shall we bump the opacity to 0.9?

Reply:
will do!